dashboard/app/views/peer_reviews/show.html.haml

```diff
@@ -0,0 +1,37 @@
+.peer-review
+  %h1 Peer Review
+  %p Please review the peer submission in the grey box below.
+
+  .submitted_readonly{style: 'padding: 8px;'}
+    %script{src: asset_path('js/initApp/initApp.js')}
+    :javascript
+      var appOptions = #{app_options.to_json};
+    = render partial: 'levels/free_response', locals: {in_level_group: true}
+
+  %br
+  %p Does this submission satisfy the requirements stated in the question?
+
+  = form_for @peer_review do |f|
+    - if @peer_review.errors.any?
+      #error_explanation
+        %h2= "#{pluralize(@peer_review.errors.count, 'error')} prohibited this peer_review from being saved:"
+        %ul
+          - @peer_review.errors.full_messages.each do |msg|
+            %li= msg
+
+    .field
+      %p
+        = f.radio_button :status, PeerReview::ACCEPTED
+        <strong>Yes</strong>, this submission <strong>meets</strong> the requirements
+      %p
+        = f.radio_button :status, PeerReview::REJECTED
+        <strong>No</strong>, this submission <strong>does not meet</strong> the requirements
+      %p
+        = f.radio_button :status, PeerReview::ESCALATED
+        %span{style: 'font-size: 0.8em;'}
+          I would like an instructor to review this submission
+    .field
+      %p Review notes:
+      = f.text_area :data, rows: 7, placeholder: 'Type your peer review here', style: 'width: 100%; box-sizing: border-box;'
```

Comment:
Let's consider using CodeMirror for the peer review. It's modularized so it should just be a small amount of JS and will give us autosave for free